### ISSUE TITLE
Running example

### ISSUE THREAD

**Issue by idkjs** [jchavarri/jsoo-react#28]
FYI, I had to change name of library in `dune` to `jsoo_react_ppx` to overcome this error:
```sh
➜  example esy
info esy 0.5.8 (using esy.json)
info building jsoo-react-ppx-esy@link-dev:../ppx
error: build failed with exit code: 1
  build log:
    # esy-build-package: building: jsoo-react-ppx-esy@link-dev:../ppx
    # esy-build-package: pwd: /Users/prisc_000/Downloads/jsoo-react-master/ppx
    # esy-build-package: running: 'refmterr' 'dune' 'build'
    File "dune", line 2, characters 7-21:
    2 |  (name jsoo-react-ppx)
               ^^^^^^^^^^^^^^
    Error: Invalid library name.
    Hint: library names must be non-empty and composed only of the following
    characters: 'A'..'Z', 'a'..'z', '_' or '0'..'9'
    error: command failed: 'refmterr' 'dune' 'build' (exited with 1)
    esy-build-package: exiting with errors above...

  building jsoo-react-ppx-esy@link-dev:../ppx
esy: exiting due to errors above
```

Thank you for doing this amazing work. I wish I was skilled enough to help you. Until then, I will run and report. Peace to you.

**Comment by jchavarri** [comment]
@idkjs thanks for reporting! I thing I renamed stuff lately and left that wrong lib name. 

I'll fix it soon, but if you feel like taking a stab at it, PRs are very welcomed! 🙂  `jsoo_react_ppx` is the name i'd use too. 👍

**Comment by idkjs** [comment]
fixed in pull request per your request,sir.

**Comment by jchavarri** [comment]
Fixed in #29.
